js/dateManipulation.js

```diff
@@ -1,11 +1,12 @@
 export const WEEKDAYS = 7
 export const HOUR_COUNT = 24
+export const LOCALE = 'us-US'
 
 export const getToday = () => new Date(Date.now())
 
-export let selectedDate = getToday()
-
 export const isSameWeek = (date1, date2) => {
+    if(!date1 || !date2) return false
```

Comment:
try not doing such fallbacks in your code - they hide errors. You will accidentally pass an undefined date somewhere and you will look for the bug for hours.
It's better for program to CRASH and BURN for unexpected errors. (I guess unless it's an airplane controlling software)

If you do have places where one or both of the dates may be undefined for good reasons, handle it there and don't call isSameWeek